pkgs/pwdsphinx/default.nix

```diff
@@ -18,14 +20,13 @@ buildPythonPackage rec {
 
   checkInputs = [ crudini ];
```

Comment:
I think this can be removed now that sed is used right?

Reply:
Yes, thanks!